### ISSUE TITLE
Update the contribute guide

### ISSUE THREAD

**Issue by zankyr** [exercism/java#2203]
The file `CONTRIBUTING.md` should be improved.

A few examples:
- missing `porting from another track` guide (the link is broken)
- old `configlet` commands no longer working (e.g. `configlet generate . --only name_of_new_exercise --spec-path path_to_problem_specifications` doesn't work for `practice` exercises).

If you agree on this issue, I've already made some fixes while working on #1810, and you can see them in the PR related to the current issue.

Advice or suggestions on these corrections are welcome.

Blocked by https://github.com/exercism/java/pull/2200

**Comment by andrerfcsantos** [comment]
Thanks for your interest in contributing, but I'm going to close this and the draft.

Exercism is fundamentally changing its contribution model and I'm not sure how relevant it is at this point to update the contributing files. Will mark this as work paused and if it's still relevant in the future, we can resume work by reopening this issue (or creating a new one).

**Comment by andrerfcsantos** [comment]
Reopening this since for the Java track we are still accepting contributions.

@zankyr Do you want me to reopen #2204 so we can keep working on it?

@vorenusCoA Where is that link you are talking about?

**Comment by vorenusCoA** [comment]
@andrerfcsantos 
Second link in line 57:
`[Contributing via GitHub](https://github.com/exercism/docs/tree/main/building/github#contributing-via-github)`

should be

`[Contributing via GitHub](https://exercism.org/docs/building/github)`

**Comment by andrerfcsantos** [comment]
@vorenusCoA There was some discussion in the forum about broken links in docs: https://forum.exercism.org/t/docs-broken-links/1038

Feel free to add a comment there and I think it's fine to open a PR in the docs repo.

**Comment by vorenusCoA** [comment]
@andrerfcsantos The links in the docs repo are ok. If you are browsing through the repo there are links that are not going to work (and that is expected).
My suggestion is to change the link present in the `CONTRIBUTING.md` so that points to the website instead to the docs repo. The reason is because if you are a new contributor (like myself) and start reading the `CONTRIBUTING.md` file you will end up in a dead link. Since the information is available in the website I think we should make a link to the website instead of the repo (the only "wrong" link I found is the one in line 57 previously mentioned)

Here is an already closed issue that I had created because I thought the links in the repo should work: https://github.com/exercism/docs/issues/413
